.github/workflows/tests.yml

```diff
@@ -0,0 +1,14 @@
+name: Run Tests
+on: [push, pull_request]
+jobs:
+  tests:
+    runs-on: ubuntu-latest
+    steps:
+    - uses: actions/checkout@v2
+    - uses: actions/setup-ruby@v1
+      with:
+        ruby-version: '2.6'
+    - name: RSpec
+      run: |
+        bundle install
+        bundle exec rake
```

Comment:
Nice 👏

Reply:
We can [speed up `bundle install` with some caching](https://github.com/actions/cache), but for the number of PRs this gets it didn't feel worth it.